service/src/main/resources/HTMLTemplate.html

```diff
@@ -1066,44 +1072,52 @@ <h3 class="heading heading--two">Respondent
     <div class="data-list__row">
       <dt class="data-list__title">Current address</dt>
       <dd class="data-list__definition">
-        {% if case_data.respondents is not empty and
-        case_data.respondents.firstRespondent is not empty and
-        case_data.respondents.firstRespondent.address is not empty %}
-          {% if case_data.respondents.firstRespondent.address.AddressLine1 is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.AddressLine1 }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.AddressLine2 is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.AddressLine2 }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.AddressLine3 is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.AddressLine3 }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.PostTown is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.PostTown }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.PostCode is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.PostCode }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.County is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.County }}</span>
-          {% endif %}
-          {% if case_data.respondents.firstRespondent.address.Country is not empty %}
-            <span class="u-comma-before">{{ case_data.respondents.firstRespondent.address.Country }}</span>
+        {% if case_data.respondents is not empty and case_data.respondents.firstRespondent is not empty %}
+          {% if case_data.respondents.firstRespondent.address is not empty %}
```

Comment:
IMO that if statement should be joined with previous one otherwise if address is empty no dash will be printed because there is no 'else' statement.

Reply:
The else for the address if is on line 1102 with the dash

Reply:
Am I understanding this correctly?